Makefile

```diff
@@ -5,7 +5,24 @@
 ## Makefile
 ##
 
-TARGET	=	glados
+TARGET		= glados
+STACK_TARGET	= GLaDOS
 
-testrun: 
-	@echo "Running tests..."
\ No newline at end of file
+all: $(TARGET)
+
+$(TARGET):
+	stack build
+	ln -sf $$(stack path --local-install-root)/bin/$(STACK_TARGET)-exe $(TARGET)
+
+clean:
+	stack clean
+
+fclean:
+	stack purge
+
+re: fclean all
+
+tests_run:
+	stack test
+
+.PHONY: $(TARGET) clean fclean re tests_run
```

Comment:
`all` should be in `.PHONY` too.
Maybe consider investigating if it's possible to get the used source files from a `stack` command to make them a proper dependency of the executable target ?
Though I think I remember Stack avoiding unecessary re-builds of files by itself anyway.

Reply:
Not putting `all` in `.PHONY` is an oversight, thanks for pointing it out

I think it's not worth the hassle of tracking dependencies of targets, let's keep this Makefile a simple wrapper around stack

Reply:
Sure.